src/frontend/web_application/src/components/MultidimensionalPi/components/PiGraph/index.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { PropTypes } from 'react';
+import './style.scss';
+
+const Grid = ({ pi, angle, axeLength }) => {
+  const axeCoordinates = [];
+  const polygonPoints = [];
+  let count = 0;
+
+  pi.map((p) => {
+    const axeName = p.name;
+    const axeX = axeLength - (axeLength * Math.sin((count * Math.PI) / 180));
+    const axeY = axeLength - (axeLength * Math.cos((count * Math.PI) / 180));
+    axeCoordinates.push({ axeName, x: axeX, y: axeY });
+    polygonPoints.push(axeX, axeY);
+    count -= angle;
+  });
+
+  return (
+    <g className="m-pi-graph__grid">
+      <polygon className="m-pi-graph__outline" points={polygonPoints.join(' ')} />
+      {axeCoordinates.map(p =>
+        <line
+          key={p.axeName}
+          className="m-pi-graph__line"
+          id={p.axeName}
+          x1={axeLength}
+          y1={axeLength}
+          x2={p.x}
+          y2={p.y}
+        />
+      )}
+    </g>
+  );
+};
+
+Grid.propTypes = {
+  pi: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
+  angle: PropTypes.number.isRequired,
+  axeLength: PropTypes.number.isRequired,
+};
+
+const Polygon = ({ pi, angle, axeLength }) => {
+  const polygonPoints = [];
+  let count = 0;
+  pi.map((p) => {
```

Comment:
"error:  Expected to return a value in this function"
I don't know how to write this function the right way, maybe @iamdey you can help ?